executor/newdeploy/newdeploy.go

```diff
@@ -45,6 +47,30 @@ const (
 	envVersion = "ENV_VERSION"
 )
 
+func (deploy *NewDeploy) getFetcherResources() (v1.ResourceRequirements, error) {
+	//TBD Hardcoded as of now, should be configurable?
+	mincpu, err := resource.ParseQuantity("10m")
+	minmem, err := resource.ParseQuantity("16Mi")
+	maxcpu, err := resource.ParseQuantity("40m")
+	maxmem, err := resource.ParseQuantity("128Mi")
```

Comment:
you can move this outside of the function (either ignore the errors as they are hardcoded anyway, or check in the `init()`, or wrap them in a helper function (like `MustParseQuantity`))